graphqlapi/src/main/java/io/stargate/graphql/schema/fetchers/ddl/TableFetcher.java

```diff
@@ -39,7 +42,7 @@ protected TableFetcher(
     }
 
     authorizationService.authorizeSchemaWrite(
-        authenticationSubject, keyspaceName, tableName, scope, SourceAPI.GRAPHQL);
+        authenticationSubject, keyspaceName, tableName, scope, SourceAPI.GRAPHQL, EMPTY_HEADERS);
```

Comment:
I'm not sure using `EMPTY_HEADERS` here is correct. In case authentication was tenant-based, do we not have to provide tenant info in this context too?

By the way, if we keep tenant ID in the `authenticationSubject` this class will not have to change at all, I believe.

Reply:
I missed that one, thx for catching it. Fixed.